infra/build/functions/target_experiment.py

```diff
@@ -115,6 +120,32 @@ def run_experiment(project_name, target_name, args, output_path,
       ],
   })
 
+  # Upload binary.
+  steps.append({
+      'name':
+          'gcr.io/cloud-builders/gsutil',
+      'entrypoint':
+          '/bin/bash',
+      'args': [
+          '-c',
+          (f'gsutil cp {local_target_path} '
+           f'{upload_reproducer_path}/{target_name} || true'),
+      ],
+  })
+
+  # Upload reproducer.
+  steps.append({
```

Comment:
See below: let's make this conditional on the flag being passed.

Reply:
Fixed